sound/soc/sof/sof-priv.h

```diff
@@ -128,6 +129,20 @@ struct snd_sof_platform_stream_params {
 	bool no_ipc_position;
 };
 
+/**
+ * struct snd_sof_module_library_info: 3rd party module library information
+ * @fw: Pointer to the 3rd party module firmware
+ * @fw_offset: Offset of the firmware within the firmware file
+ * @name: 3rd party module firmware name
+ * @id: ID of the module library
```

Comment:
what is the ID here? 

it's not a UUID, it's not an index in the module array. Not sure what this represents.